src/main.rs

```diff
@@ -252,9 +276,86 @@ async fn main() {
                 println!("Link data:\n{:?}", link.unwrap());
             }
         },
+
+        Command::File(file) => match &file {
+            FileCommand::Insert { file_path, content } => {
+                let file_path = file_path.as_ref().unwrap();
+                let atomic_file = arklib::AtomicFile::new(file_path).unwrap();
+
+                if let Some(content) = content {
+                    modify(&atomic_file, |_| content.as_bytes().to_vec())
+                        .unwrap();
+                }
+            }
+
+            FileCommand::List { file_path, all } => {
```

Comment:
```
 FileCommand::List { storage, all } => {
     let file_path = ...
```

Let's support also passing a storage name with the following logic:
* if `storage` is a valid path, then use it to initialize `file_path`
* if `storage` isn't a valid path, then check if it's one of "official" storage names (path constants from `arklib`); initialize `file_path` with the path of such a storage (must be `.ark/...`)

Reply:
And, if `storage` is absent, then we list all storages in `.ark` but we don't print all versions for each storage, only brief information. For simplicity, we could just use path constants from `arklib` (`lib.rs`). We'll need to update this `list` command manually when new storages are added.

In this mode, we should display for each storage:
* current latest version number
* app id of the latest version
* information about forks if such exist